modules/ROOT/pages/amazon/amazon-dynamodb-connector-reference.adoc

```diff
@@ -1,1190 +1,1167 @@
-* DYNAMODB:LimitExceededException
-* DYNAMODB:ProvisionedThroughputExceededException
-* DYNAMODB:ResourceInUseException
-* DYNAMODB:ResourceNotFoundException
-* DYNAMODB:RETRY_EXHAUSTED
-* DYNAMODB:Unknown
+* <<config>> 
+
+==== Throws
+* DYNAMODB:ACCESS_DENIED_EXCEPTION 
+* DYNAMODB:CONDITIONAL_CHECK_FAILED_EXCEPTION 
+* DYNAMODB:CONNECTIVITY 
+* DYNAMODB:INCOMPLETE_SIGNATURE_EXCEPTION 
+* DYNAMODB:ITEM_COLLECTION_SIZE_LIMIT_EXCEEDED_EXCEPTION 
+* DYNAMODB:LIMIT_EXCEEDED_EXCEPTION 
+* DYNAMODB:MISSING_AUTHENTICATION_TOKEN_EXCEPTION 
+* DYNAMODB:PROVISIONED_THROUGHPUT_EXCEEDED_EXCEPTION 
+* DYNAMODB:RESOURCE_IN_USE_EXCEPTION 
+* DYNAMODB:RESOURCE_NOT_FOUND_EXCEPTION 
+* DYNAMODB:RETRY_EXHAUSTED 
+* DYNAMODB:THROTTLING_EXCEPTION 
+* DYNAMODB:UNKNOWN 
+* DYNAMODB:UNRECOGNIZED_CLIENT_EXCEPTION 
+* DYNAMODB:VALIDATION_EXCEPTION 
 
 
 [[batchGetItem]]
-== Batch Get Item
-
+=== Batch Get Item
 `<dynamodb:batch-get-item>`
 
-The BatchGetItem operation returns the attributes of one or more items from one or more tables. You identify requested items by the primary key.
 
-=== Parameters
+ The BatchGetItem operation returns the attributes of one or more items from one or more tables. You identify requested items by primary key. See the Amazon http://docs.aws.amazon.com/amazondynamodb/latest/APIReference/API_BatchGetItem.html[API Reference] for this operation. 
+
 
-[%header%autowidth.spread]
+==== Parameters
+[%header,cols="20s,20a,30a,20a,10a"]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | |x
-| Request Items a| Object |  A map of one or more table names and, for each table, a map that describes one or more items to retrieve from that table. |  |x
+| Configuration | String | The name of the configuration to use. | | x
+| Request Items a| Object |  A map of one or more table names and, for each table, a map that describes one or more items to retrieve from that table. |  | x
 | Return Consumed Capacity a| Enumeration, one of:
 
 ** INDEXES
 ** TOTAL
-** NONE |  Determines the level of detail about provisioned throughput consumption that is returned in the response. |  |
-| Target Variable a| String |  The name of a variable where the output of this operation is stored |  |
-| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the target variable. |  `#[payload]` |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever |  A retry strategy in case of connectivity errors |  |
+** NONE |  Determines the level of detail about provisioned throughput consumption that is returned in the response. |  | 
+| Target Variable a| String |  The name of a variable to store the operation's output. |  | 
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  `#[payload]` | 
```

Comment:
Same wording as before: An expression to evaluate against the operation's output. The expression's outcome is stored in the target variable.